doc/testing-your-application.rst

```diff
@@ -18,10 +17,17 @@ By having your test classes extend one of the test case classes, you get:
   reverse proxy server. See reverse proxy specific sections for details;
 * convenience methods for executing HTTP requests to your application:
   ``$this->getHttpClient()`` and ``$this->getResponse()``;
-* custom assertions ``assertHit`` and ``assertMiss`` for validating a cache hit/miss.
+* custom assertions ``assertHit`` and ``assertMiss`` for validating a cache
+  hit/miss.
 
 The recommended way to configure the test case is by setting constants
-in your `phpunit.xml`. Alternatively, you can override the getter methods.
+in your ``phpunit.xml``. Alternatively, you can override the getter methods.
+
+You will need to run a web server to provide the PHP application you want to
+test. The test cases only handle running the caching proxy. With PHP 5.4 or
+newer, the easiest is to use the PHP built in web server. See the
+``WebServerListener`` class in ``tests/Functional`` and how it is registered in
+``phpunit.xml.dist``.
```

Comment:
actually we could make this easily reusable. but moving it might break things for people who already used it, so created a v2 task: https://github.com/FriendsOfSymfony/FOSHttpCache/issues/206